app/main.py

```diff
@@ -43,4 +43,8 @@ def bring_slippers() -> None:
 
 
 def feed_animals(animals: list[Animal]) -> int:
-    return sum([animal.feed() for animal in animals])
+    sum_of_feed_animals = 0
+    for animal in animals:
+        sum_of_feed_animals += Animal.feed(animal)
+
+    return sum_of_feed_animals
```

Comment:
there was no need to use a for loop. You could simply remove the square brackets [] to use a generator expression